activesupport/test/time_zone_test.rb

```diff
@@ -8,6 +8,12 @@
 class TimeZoneTest < ActiveSupport::TestCase
   include TimeZoneTestHelpers
 
+  def setup
+    # Clear memoized time zone data on TimeZone class
+    ActiveSupport.send(:remove_const, :TimeZone) if ActiveSupport.const_defined?(:TimeZone)
```

Comment:
Still not happy with this - I'd rather we add a nodoc public `clear` method on `AS::TimeZone` to explicitly clear the cache and then use that in the test.